build.gradle.kts

```diff
@@ -53,7 +53,7 @@ buildTypes {
         tasks(
             "classes", "doc:checkstyleApi", "codeQuality", "allIncubationReportsZip",
             "docs:check", "distribution:checkBinaryCompatibility", "javadocAll",
-            "architectureTest:test")
+            "architectureTest:test", "toolingApi:toolingApiShadedJar")
```

Comment:
This is for the Gradleception build, right?

Reply:
Yeah, Gradleception build and some extra verification in `sanityCheck`.